packages/filebrowser/src/browser.ts

```diff
@@ -180,6 +236,51 @@ export class FileBrowser extends SidePanel {
     }
   }
 
+  /**
+   * Whether to show a text box to filter files by name.
+   */
+  get showFileFilter(): boolean {
```

Comment:
Note for the next revision: we should probably think about the API surface; at first glance it appears that having both `showFileFilter` and `toggleFileFilter` as public may not be necessary. I think that once you implement the suggestion of using command with `isToggled` for reflecting the pressed status the need for `toggleFileFilter` will disappear.